HISTORY.rst

```diff
@@ -6,6 +6,11 @@ History
 0.12.0 (unreleased)
 ===================
 
+* **Breaking Change**: Change in ``AnnulusDescription`` to support different types of annulus equipment. Now ``AnnulusDescription``` has an attribute ``AnnulusEquipmentDescription``, which holds a dict that can contain multiple different equipment types, for which the current available options are:
+
+  - ``LeakEquipmentDescription``;
```

Comment:
@oliveira-mauricio 
Don't you think we should to start to mention explicitly how this affects alfacase files in the history?

Reply:
I agree. 👍

Reply:
Like an example? I guess it's ok.

Reply:
Maybe something like this?

* Here is an example of how to update previous .alfacase files with annulus equipment:

**Before**
```
  annulus:
    gas_lift_valve_equipment:
      Gas Lift Valve (Well 1 > Annulus) 1:
        position:
          value: 100.0
          unit: m
        ...  # Other properties
```

**After**  

```
  annulus:
    equipment:
      gas_lift_valves:
        Gas Lift Valve (Well 1 > Annulus) 1:
          position:
            value: 100.0
            unit: m
          ...  # Other properties
```

Reply:
Great!

Reply:
And maybe add that on the previous release entry too?

* Here is an example of how to update previous .alfacase files with trends:

** Before**

```
  trends:
  - curve_names:
    - timestep
    location: not_defined
  - curve_names:
    - flow pattern
    - pressure
    element_name: Conn 1
    ... # Other properties
```

** After **

```   
  trends:
    positional_pipe_trends:
    - curve_names:
      - flow pattern
      - pressure
      element_name: Conn 1
      ... # Other properties
    overall_pipe_trends: []
    global_trends:
    - curve_names:
      - timestep
    equipment_trends: []
    separator_trends: []
```